wallet-new/src/Cardano/Wallet/Kernel/DB/InDb.hs

```diff
@@ -58,6 +58,10 @@ instance SafeCopy (InDb Core.Coin) where
   getCopy = error "TODO: getCopy for (InDb Core.Coin)"
   putCopy = error "TODO: putCopy for (InDb Core.Coin)"
 
+instance SafeCopy (InDb Core.SlotId) where
+  getCopy = error "TODO: getCopy for (InDb Core.SlotId)"
+  putCopy = error "TODO: putCopy for (InDb Core.SlotId)"
```

Comment:
@edsko told me but I have forgot: which is the name of the precise ticket where we are going to "fill" all this stubbed implementations?

Reply:
we could do it as part of this  "[CBR-300] Address all the TODOs left dangling after CBR-238"

Reply:
agreed, it should happen asap, I'll get onto it before rollback for that matter https://iohk.myjetbrains.com/youtrack/issue/CBR-305

Reply:
As per the meeting yesterday, let's give rollback a higher priority.